recipe/meta.yaml

```diff
@@ -27,7 +27,8 @@ requirements:
   build:
     - python                                 # [build_platform != target_platform]
     - cross-python_{{ target_platform }}     # [build_platform != target_platform]
-    - nanobind                               # [build_platform != target_platform]
+    - nanobind >=2                           # [build_platform != target_platform]
```

Comment:
pyproject.toml declares:

```toml
"nanobind >=1.9.2"
```

Reply:
Oof, that's a package bug. The stub generator will fail on anything less than 2, and it's a mandatory part of the build.

Reply:
Will be fixed by https://github.com/inducer/islpy/pull/170

Reply:
Should this now become `>=2.7`?